tests/acceptance/requirements_deb.txt

```diff
@@ -0,0 +1,59 @@
+autoconf
+automake
+awscli
+bash
+bc
+build-essential
+chrpath
+cpio
+curl
+device-tree-compiler
+diffstat
+docker.io
+e2tools
+gawk
+gdisk
+git
+gnupg2
+jq
+kmod
+kpartx
+libaio-dev
+libbluetooth-dev
+libbrlapi-dev
+libbz2-dev
+libfdt-dev
+libffi-dev
+libglib2.0-dev
+liblzma-dev
+liblzo2-dev
+libpixman-1-dev
+libsdl1.2-dev
+libssl-dev
+libvirt-bin
+libxml2-dev
+libxslt-dev
+libyaml-dev
+locales
+nfs-client
+nfs-common
+nodejs
+pass
+pkg-config
+psmisc
+python-dev
+python3-pip
+qemu-kvm
+qemu-system-arm
+qemu-system-x86
+qemu-utils
+s3cmd
+screen
+sudo
+sysbench
+sysstat
+texinfo
+wget
+zlib1g-dev
+rustc
```

Comment:
Unrelated to this PR, but I don't fully understand why the rust compiler is needed, like it seems to me that prebuilt binaries are available for `x86` [here](https://pypi.org/project/cryptography/#files)

Reply:
you might find [this](https://github.com/mendersoftware/mender-test-containers/pull/49/files/9930baa667b772e088a2d29ddcce6b41f7ec369e#diff-10213bb36066710156ec0a50127d6bd0e8117030ea1a6bc2357ae0d5e55b8e31) relevant - TLDR: ubuntu package updates are just too damn slow for us 😉

Reply:
That explains it!
I find that I'm not missing Ubuntu much

Reply:
Soon (in theory this month) the next yocto release will be out, then we migrate, then we deprecate `warrior`, and *then* we update the base image to next Ubuntu LTS. Be patient my friends :)

Reply:
Sorry to burst the bubble, but we only follow LTS releases now, and the next one is not out until April 2022. :-)

Dunfell will live on for one more year! And warrior, well maybe also one year, but it is not LTS, so we might want to discuss that.